source/codegen/templates/service_helpers.mako

```diff
@@ -213,13 +221,17 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
 %>\
 %     if common_helpers.is_struct(parameter):
       std::vector<${underlying_param_type}> ${parameter_name}(${size}, ${underlying_param_type}());
+%     elif underlying_param_type == 'ViBoolean':
+      std::vector<${underlying_param_type}> ${parameter_name}(${size}, ${underlying_param_type}());
+      response->mutable_${parameter_name}()->Resize(${size}, 0);
```

Comment:
The `Resize` on the response field is only needed when passing in the raw response array to the function. Since you're not doing so, this should follow the same code path as the custom structs.

Reply:
I didn't understand this comment. I thought `Resize` was to resize the array the output array to correct size. I have removed the `Resize` call and it seems to work, just wanted to understand the comment better.

Reply:
@jonlugoatx looks like you missed replying to this comment.